bal_tools/models.py

```diff
@@ -154,3 +154,21 @@ def validate_model(cls, values):
             if field in values:
                 values[field] = cls.str_to_decimal(values[field])
         return values
+
+      
+class PoolData(BaseModel):
+    id: str
+    symbol: str
+    dynamicData: dict
```

Comment:
i think this can be dropped too. we are always filtering on `pool_data.dynamicData['swapEnabled']`, so the attribute is useless no?

Reply:
this is separate from whatever the query_all_pools method is doing, the model is for modeling the data from the api. in this case, each element in the `poolGetPools` query

Reply:
hmm ok, i guess we could pop it before returning at the end of `query_all_pools` but it is not that important